travis-ymls/maxmin.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
```

Comment:
the header for the travis.yml file is missing, please refer to https://github.com/ppc64le/build-scripts/blob/master/travis-ymls/README.md

Reply:
added